mora/service/mapping.py

```diff
@@ -5,7 +5,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
-
+from mora.service import common, keys
```

Comment:
These could be relative imports — and you've duplicated the import of `common`. (Also, see [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) on imports — modules should be one per line.)